.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,36 +2,44 @@
 name: Bug report
 about: Create a report to help us improve
 title: 'auro-lockup: [issue summary]'
-labels: 'Type: Bug'
+labels: 'Type: Bug, not-reviewed'
 assignees: blackfalcon
 ---
 
 ## Describe the bug
-A clear and concise description of what the bug is.
+
+<!-- A clear and concise description of what the bug is. -->
 
 ## To Reproduce
+
 Steps to reproduce the behavior:
+
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
```

Comment:
Numbering got screwed up here

Reply:
Nope, intended. You can do this and markdown will convert it to the correct numbers.